package.nls.json

```diff
@@ -60,5 +60,6 @@
     "azFunc.disconnectRepo": "Disconnect from Repo...",
     "azFunc.createSlot": "Create Slot...",
     "azFunc.swapSlot": "Swap Slot...",
-    "azFunc.advancedCreationDescription": "Enables advanced creation of function apps, which bypasses several defaults and prompts instead."
+    "azFunc.advancedCreationDescription": "Enables advanced creation of function apps, which bypasses several defaults and prompts instead.",
```

Comment:
nit: "bypasses several defaults and prompts"... by then my mind assumes prompts is a noun and has to reread the sentence.  suggest: "restores several prompts instead of using defaults"??  There's probably something better.

Reply:
I see what you mean about the noun. Messed with the message a bit